concrete/src/Job/Service.php

```diff
@@ -0,0 +1,114 @@
+<?php
+namespace Concrete\Core\Job;
+
+use Concrete\Core\Application\Application;
+
+class Service
+{
+    protected $app;
+
+    public function __construct(Application $app)
+    {
+        $this->app = $app;
+    }
+
+    /**
+     * @param $jHandle
+     * @param $pkg
+     *
+     * @return false|Job
+     */
+    public function installByPackage($jHandle, $pkg)
```

Comment:
Could you elaborate on this a bit more? In your code example it looks like the job is installing itself.